File: app/src/main/java/com/example/newbiechen/ireader/utils/IOUtils.java
package com.example.newbiechen.ireader.utils;

import java.io.Closeable;
import java.io.IOException;

/**
 * Created by wangzhen on 17-5-11.
 */

public class IOUtils {

    public static void close(Closeable closeable){
        if (closeable == null) return;
        try {
            closeable.close();
        } catch (IOException e) {
            e.printStackTrace();
            //close error
        }
    }
}

File: app/src/main/java/com/wangzhen/reader/utils/IOUtils.kt
package com.wangzhen.reader.utils

import java.io.Closeable
import java.io.IOException

/**
 * IOUtils
 * Created by wangzhen on 2023/4/13
 */
object IOUtils {
    @JvmStatic
    fun close(vararg closeables: Closeable) {
        closeables.forEach {
            try {
                it.close()
            } catch (ignored: IOException) {
            }
        }
    }
}